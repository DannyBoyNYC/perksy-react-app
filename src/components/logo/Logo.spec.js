const React = require('react');

const Logo = require('./index').default;

describe('<Logo.* />', () => {
  let component;
  const requiredProps = {};

  describe('<Logo.Black />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(<Logo.Black {...requiredProps} />, 'Logo');
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });

  describe('<Logo.White />', () => {
    beforeEach(() => {
      component = shallowUntilTarget(<Logo.White {...requiredProps} />, 'Logo');
    });

    it('should render correctly with all required props', () => {
      expect(component).toMatchSnapshot();
    });
  });
});
