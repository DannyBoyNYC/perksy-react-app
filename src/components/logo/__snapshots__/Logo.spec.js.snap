// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo.* /> <Logo.Black /> should render correctly with all required props 1`] = `
<div
  className="Logo-root-0-1-1"
>
  <img
    alt="Perksy black logo"
    className="Logo-image-0-1-2"
    src="/static/logo_black.svg"
  />
</div>
`;

exports[`<Logo.* /> <Logo.White /> should render correctly with all required props 1`] = `
<div
  className="Logo-root-0-1-1"
>
  <img
    alt="Perksy white logo"
    className="Logo-image-0-1-2"
    src="/static/logo_white.svg"
  />
</div>
`;
